src/java/org/apache/cassandra/tools/BulkLoader.java

```diff
@@ -67,9 +73,11 @@ public static void load(LoaderOptions options) throws BulkLoadException
                         handler,
                         options.connectionsPerHost,
                         options.targetKeyspace);
-        DatabaseDescriptor.setStreamThroughputOutboundMegabitsPerSec(options.throttle);
-        DatabaseDescriptor.setInterDCStreamThroughputOutboundMegabitsPerSec(options.interDcThrottle);
-        StreamResultFuture future = null;
+        DatabaseDescriptor.setStreamThroughputOutboundMegabitsPerSec(Double.valueOf(options.throttleMegabits).longValue());
+        DatabaseDescriptor.setInterDCStreamThroughputOutboundMegabitsPerSec(Double.valueOf(options.interDcThrottleMegabits).longValue());
```

Comment:
`DatabaseDescriptor.setStreamThroughputOutboundMegabitsPerSec` and `DatabaseDescriptor.setInterDCStreamThroughputOutboundMegabitsPerSec` take int as a param. So we need to cast it. In your previous comment you mentioned that in `public static void setStreamThroughputOutboundMegabitsPerSec(int value)` `value` should stay as `int`

Reply:
You are right, that one is called directly in nodetool and we should keep it int as otherwise we will introduce a bug by extending to long or double.